packages/mds-test-data/index.ts

```diff
@@ -303,7 +305,7 @@ function makeStatusChange(device: Device, timestamp: Timestamp, provider_name =
   }
 }
 
-function makeTrip(device: Device, provider_id = TEST_UUID, provider_name = 'test_provider'): Trip {
+function makeTrip(device: Device, provider_id = TEST1_PROVIDER_ID, provider_name = 'test_provider'): Trip {
```

Comment:
I think `provider_name` here could be a call to `providerName`, something like `provider_name = providerName(TEST1_PROVIDER_ID)`

Reply:
Fixed.